<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Huffman Archiver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="compressButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>COMPRESS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decompressButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>DECOMPRESS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="zip">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Zipping...</string>
    </property>
   </widget>
   <widget class="QLabel" name="unzip">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Unzipping...</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="zipDone">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Done!</string>
    </property>
   </widget>
   <widget class="QLabel" name="unzipDone">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Done!</string>
    </property>
   </widget>
   <zorder>zipDone</zorder>
   <zorder>compressButton</zorder>
   <zorder>decompressButton</zorder>
   <zorder>quitButton</zorder>
   <zorder>unzip</zorder>
   <zorder>unzipDone</zorder>
   <zorder>zip</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
